sa/sa.go

```diff
@@ -1299,6 +1298,18 @@ func (ssa *SQLStorageAuthority) NewOrder(ctx context.Context, req *corepb.Order)
 		}
 	}
 
+	for _, name := range req.Names {
+		reqdName := &requestedNameModel{
+			OrderID:      order.ID,
+			ReversedName: ReverseName(name),
+		}
+		err = tx.Insert(reqdName)
+		if err != nil {
+			err = Rollback(tx, err)
+			return nil, err
```

Comment:
nit: the above assignment and a few below aren't necessary, can just be `return nil, Rollback(tx, err)`. Not really required but a general style thing.